### PR TITLE
41 styles

### PR DESCRIPTION
**Pull request by RainbowGiantSquid**
sort of working
3:59
don't know what the elements are getting so spread on the last page of the progress bar

### CODE REVIEW

File: app/assets/stylesheets/components/_progress.scss [hunk 1]
```diff
@@ -61,10 +63,173 @@
...
+            width: 1rem;
+            height: rem-calc(70);
+
+            position: absolute;
+            z-index: 1;
+          }
+          .sv_progress-buttons__page-title {
+            all: inherit;
+            margin-left: -3rem;
+            background: transparent;
```
Comment (yndajas) [inline app/assets/stylesheets/components/_progress.scss:213]: Rule order

File: app/assets/stylesheets/components/_progress.scss [hunk 2]
```diff
@@ -61,10 +63,173 @@
...
+          }
+        }
+
+        &.sv_progress-buttons__list-element--current ~ li {
+          padding-left: 4rem;
+          background-color: $grey-mdlight;
+          font-size: rem-calc(20);
+          font-weight: bold;
+          font-family: $body-font;
+          color: $grey-xdark;
```
Comment (yndajas) [inline app/assets/stylesheets/components/_progress.scss:179]: Rule order

File: app/assets/stylesheets/components/_progress.scss [hunk 3]
```diff
@@ -61,10 +63,173 @@
...
+
+          &::after {
+            all: unset;
+          }
+        }
+
+        &.sv_progress-buttons__list-element--current {
+          all: unset;
+          position: inherit;
+          z-index: 3;
```
Comment (yndajas) [inline app/assets/stylesheets/components/_progress.scss:149]: Rule order

**Review by yndajas — APPROVED** [review]
A few rule order things but looks good so I think this can just be merged for now

**Comment by sergiomarrocoli** [comment]
functions well. goes a bit crazy when the form is short, leaving white space between the bar and the form:
image

**Comment by RainbowGiantSquid** [comment]
> functions well. goes a bit crazy when the form is short, leaving white space between the bar and the form: image

Oh yes sorry - just pushed fix for this.

**Pull request merged**